src/getProjectId.js

```diff
@@ -21,10 +22,19 @@ module.exports = function(options, allowNull = false) {
 
     if (aliasCount === 0) {
       throw new FirebaseError(
-        "No project active. Run with " +
-          clc.bold("--project <projectId>") +
-          " or define an alias by\nrunning " +
-          clc.bold("firebase use --add"),
+        "No currently active project.\n" +
+          "To run this command, you have two options:\n" +
+          "- Run this command with " +
+          clc.bold("--project <project-id | project-alias>") +
+          ".\n" +
+          "- Set an active project by running " +
+          clc.bold("firebase use --add") +
+          ", then rerun this command.\n" +
+          "To list all the Firebase projects to which you have access, run " +
+          clc.bold("firebase projects:list") +
+          marked(
+            ".\nTo learn about active projects for the CLI, visit https://firebase.google.com/docs/cli#project_aliases."
```

Comment:
Why is there a period at the beginning of this line?

Also, sometimes terminals are annoying and periods attached to URLs will come along with them - can we omit the last period? cc @rachelsaunders

Reply:
ya, I don't think anyone will misunderstand anything if the period is left off :-)

Reply:
Removed the trailing period. The weird period at the start followed by the newline is to end the previous sentence, but I'll split it so that the code reads more like what the user sees.